tests/dummy/app/controllers/form.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  settings: {
```

Comment:
Do you think this could be adjusted to make the properties passed in via the template?  I'm not in love having to define configuration objects on controllers.  something like `{{ui-for inline=true}}`